src/common/feedGenerator.ts

```diff
@@ -218,7 +218,6 @@ export const applyFeedWhere = (
   allowPrivateSources = true,
 ): SelectQueryBuilder<Post> => {
   let newBuilder = builder
-    .andWhere(`${alias}.deleted = false`)
     .andWhere(`${alias}."sourceId" != :unknown`, { unknown: UNKNOWN_SOURCE })
```

Comment:
This is not needed any more as well right?

Reply:
Oh, right. Will do a whole cleanup for any existing implementations, tomorrow.

Reply:
Did clean-ups now, and just kept the source condition since as discussed, we will remove same condition from the View condition.